src/test/java/net/datafaker/datafaker_gen/DatafakerGenTest.java

```diff
@@ -0,0 +1,38 @@
+package net.datafaker.datafaker_gen;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class DatafakerGenTest {
+
+    @Test
+    void generateJsonWith10LineAndFileOutput() throws IOException {
+
+        DatafakerGen.main(new String[]{"-f", "xml", "-n", "10", "-sink", "textfile", "-oc", "output.yaml"});
```

Comment:
Wouldn't it be better to have a `@ParameterizedTest` with `@ValueSource` where input is a command like `-f xml -n 10 -sink textfile -oc output.yaml`
and then 
```java
DatafakerGen.main(input.split("\\s+"));
```

Reply:
Added ParameterizedTest for this method